packages/jest-each/src/table/template.ts

```diff
@@ -70,11 +70,8 @@ const replaceKeyPathWithValue = (template: Template) => (
   return title.replace(match, pretty(value, {maxDepth: 1, min: true}));
 };
 
-const getPath = (
-  template: Template | any,
-  [head, ...tail]: Array<string>,
-): any => {
+const getPath = (template: Template, [head, ...tail]: Array<string>): any => {
```

Comment:
any way to have it not return `any`? `unknown` is preferred

Reply:
shouldn't it return `Template | undefined`?

Reply:
I will update this with an overload to return the actual nested type, and unknown if it cant determine. I will overload up to 5 nested levels. This also may be useful to move it to `jest-utils` in another PR.

```ts
const test = getPath({a: {b: 2}}, 'a'); // return type:{b:number}
const test2 = getPath({a: {b: 2}}, ['a']); // return type:{b:number}
const test3 = getPath({a: {b: 2}}, ['a', 'b']); // return type:number
const test4 = getPath({a: {b: 2}}, ['a', 'bc']); // return type:unknown
const test5 = getPath({}, 'a'); // return type:unknown
```